packages/create-schemas/src/bin.ts

```diff
@@ -29,3 +32,55 @@ function printConfigurationErrors(error: ZodError) {
     });
     console.log("Use --help to see available options.");
 }
+
+function printWatchEvents(initialConfig: ResolvedConfig, watcher: Watcher) {
```

Comment:
If i understand correctly, we start the watcher as a CLI app and then we keep the process alive in the terminal? We then output events on the terminal

Reply:
Pretty much, yes.

When `--watch` is enabled, the process becomes persistent and won't exit unless the user manually aborts it with <kbd>CTRL</kbd> + <kbd>C</kbd>. `printWatchEvents()` prints a message whenever a new build is started or completed.